c2cgeoportal/scripts/c2ctool.py

```diff
@@ -155,8 +162,11 @@ class C2cTool:
 
     def print_step(self, step, intro="To continue type:"):
         print(intro)
-        print(_colorize("%s upgrade %s %s --step %i", YELLOW) % (
+        print(_colorize("%s upgrade %s%s%s %s --step %i", YELLOW) % (
             "%s/c2ctool" % self.venv_bin,
+            '--windows ' if self.options.windows else '',
+            '--git-remote '+self.options.git_remote+' '  \
+                if self.options.git_remote != 'origin' else '',
```

Comment:
Travis will not be happy with all those single quotes, and probably also about the missing spaces around the `+`...